app/lib/dns.server.ts

```diff
@@ -53,7 +78,7 @@ export const route53Client = new Route53Client({
   credentials: credentials(),
 });
 
-export const createHostedZone = async (domain: string) => {
+export async function createHostedZone(domain: string) {
```

Comment:
Just for my own curiosity. Does this make a difference?

Reply:
It has to do with [JS hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting) and how a `function` vs. a `const` appears in the current scope.  I need to call this function before it gets declared, so I'm relying on it being hoisted to the top of the scope.